packages/schema-poller/src/queries/index.ts

```diff
@@ -0,0 +1 @@
+export { getDataSummary } from './dataSummary';
```

Comment:
I wanted to expose this function so that we can use it in the vscode extension package to list the labels and relationship types in the tree view

I don't know that it really makes sense for the vscode extension to consume a package called 'schema-poller' when all it's doing is running a query.. perhaps there is a better place for these queries so that we can share them between packages.. but I wonder if this is a premature optimisation at this point

Reply:
Otherwise using vscode APIs I think would make sense?

Reply:
yes, I think this is a good change.. the way I've approached this is that the connections side panel and the language server may operate somewhat independently of one another - the connection panel allowing you to browse the labels and relationships of different connections.. but I think isolating it so that you may only browse the properties of the connection you're connected to makes sense

Reply:
okay - I have made this change

I have established an EventEmitter (on the poller.. I did try to establish it in the connection, but communication wasn't picking up properly.. I'll try again in the morning..) that I am using to communicate events from the poller to the language server, the language server then uses a communication mechanism to communicate with the extension i.e. poller -> language server -> vscode extension

I opted for the additional hop so we didn't need to expose the vscode extension to the poller class

--

the reason I did this in the first place, and it's not right in hindsight, is that I was thinking we could have the connection panel independent of the language server connection.. i.e. if you had multiple connections you could browse their labels and nodes regardless of being connected to it

anyway, this is better.. and it makes more sense